csunplugged/utils/convert_heading_tree_to_dict.py

```diff
@@ -0,0 +1,45 @@
+"""Module used for converting Verto heading tree data.
+
+The Verto result object is a tuple containing NamedTuples, however this
+must be converted to a dictionary to be stored in a HStoreField Postgres
+database field.
+"""
+
+
+def convert_heading_tree_to_dict(heading_tree_tuples):
+    """Convert tuple heading tree to dictionary.
+
+    Args:
+        heading_tree_tuples: The heading tree from a Verto conversion.
+
+    Returns:
+        Dictionary of heading tree, or None if None provided as tree.
+    """
+    if heading_tree_tuples is None:
+        return None
+    else:
+        root_nodes = []
+        for heading_node in heading_tree_tuples:
+            root_nodes.append(convert_heading_node(heading_node))
+        return root_nodes
```

Comment:
`return [convert_heading_node(heading_node) for heading_node in heading_tree_tuples]`